locales/en.yml

```diff
@@ -21,27 +41,162 @@ pico:
+          emptyName: "The name cannot be empty."
+          noCommand: "No command specified."
+          noDescription: "No description specified."
+          noName: "No name specified."
+        info:
+          askCommand: "Command: "
+          askDescription: "Description: "
+          askName: "Name: "
+          taskCreated: "Task has been created successfully."
+    list:
+      description: "List all tasks"
+      noTasks: "No tasks found."
+      info:
+        foundTasks: "Found %d tasks."
+      options:
+        json:
+          description: "output the tasks in JSON format"
+    new:
+      description: "Create a new task"
+      options:
+        assumeYes:
+          description: "assume yes for all prompts"
+        at:
+          description: "schedule the task to execute at a specific time (hh:mm)"
+        command:
+          description: "specify the command to execute"
+        description:
+          description: "specify the task's description"
+        every:
+          description: "schedule the task to execute every X time (minutes, hours, days)"
+        name:
+          description: "specify the task's name"
+        needConfirm:
+          description: "ask for confirmation before executing the task"
+        onBattery:
+          description: "execute the task only when the system is on battery"
+        onCharge:
+          description: "execute the task only when the system is charging"
+        onConditionCommand:
+          description: "execute the task on condition command"
+        onCpuTemp:
+          description: "execute the task when CPU temperature is higher than 60°C"
+        onCpuUsage:
+          description: "execute the task when CPU usage is higher than X%"
+        onDeviceConnect:
+          description: "execute the task when a device is connected"
+        onDeviceDisconnect:
+          description: "execute the task when a device is disconnected"
+        onDisconnect:
+          description: "execute the task when the system is disconnected from the network"
+        onFullBattery:
+          description: "execute the task when the system is fully charged"
+        onHighCpuUsage:
+          description: "execute the task when CPU usage is higher than X%"
+        onHighInternetUsage:
+          description: "execute the task when internet usage is higher than 500Kb/s"
+        onHighMemoryUsage:
+          description: "execute the task when memory usage is higher than 50%"
+        onInternetUsage:
+          description: "execute the task when internet usage is higher than XKb/s"
+        onLowBattery:
+          description: "execute the task when the system is low on battery (20%)"
+        onMemoryUsage:
+          description: "execute the task when memory usage is higher than X%"
+        onNetwork:
+          description: "execute the task when the system is connected to the network"
+        onProcess:
+          description: "execute the task when a process is running"
```

Comment:
Maybe `execute the task when a process starts`?

Reply:
```suggestion
          description: "execute the task when a process starts"
```